csharp/autobuilder/Semmle.Autobuild.CSharp/DotNetRule.cs

```diff
@@ -276,17 +243,14 @@ private static BuildScript GetInstalledRuntimesScript(IBuildActions actions, str
         /// hence the need for CLR tracing), by adding a
         /// `/p:UseSharedCompilation=false` argument.
```

Comment:
Shouldn't the comment regarding `.NET Core >= 3` be removed? We're not checking the version any longer.

Reply:
Good spot.